src/main/java/models/Dealer.java

```diff
@@ -0,0 +1,25 @@
+package models;
+
+
+import java.io.Serializable;
+import java.util.ArrayList;
+
+
+public class Dealer extends Person implements Serializable{
+
+
+	//inherit from constructor
+	public Dealer(){
+		super();
+	}
+
+
+	//set the bet to what the dealer now wants
+	public void makeBet(int newBet)
```

Comment:
The dealer doesn't bet. They have an unlimited bank so there's no reason to keep track of their money.